src/infrastructure/database/utils/prepared-statements.ts

```
@@ -179,25 +217,35 @@ export class PreparedStatementManager {
           }
           const duration = Date.now() - startTime;
-          QueryLoggerService.logQuery(`PREPARED ${name}: ${query}`, params, duration, 'sqlite');
+          QueryLoggerService.logQuery(
+            `PREPARED ${name}: ${query}`,
+            params,
+            duration,
+            'sqlite'
+          );
 
           // Update usage statistics
           if (this.cache[name]) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)